src/main/java/com/google/api/codegen/discovery/transformer/go/GoSampleMethodToViewTransformer.java

```diff
@@ -0,0 +1,211 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.discovery.transformer.go;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.api.codegen.discovery.config.FieldInfo;
+import com.google.api.codegen.discovery.config.MethodInfo;
+import com.google.api.codegen.discovery.config.SampleConfig;
+import com.google.api.codegen.discovery.transformer.SampleMethodToViewTransformer;
+import com.google.api.codegen.discovery.transformer.SampleNamer;
+import com.google.api.codegen.discovery.transformer.SampleTransformerContext;
+import com.google.api.codegen.discovery.transformer.SampleTypeTable;
+import com.google.api.codegen.discovery.viewmodel.SampleAuthView;
+import com.google.api.codegen.discovery.viewmodel.SampleFieldView;
+import com.google.api.codegen.discovery.viewmodel.SamplePageStreamingView;
+import com.google.api.codegen.discovery.viewmodel.SampleView;
+import com.google.api.codegen.util.Name;
+import com.google.api.codegen.util.SymbolTable;
+import com.google.api.codegen.util.go.GoTypeTable;
+import com.google.api.codegen.viewmodel.ViewModel;
+import com.google.protobuf.Method;
+
+public class GoSampleMethodToViewTransformer implements SampleMethodToViewTransformer {
```

Comment:
If there are any functions in here that are mostly the same across languages, create a cross-language transformer and call it from the language-specific transformers.